include/geom/bounding_box.h

```diff
@@ -116,6 +116,17 @@ class BoundingBox : public std::pair<Point, Point>
    */
   bool contains_point (const Point &) const;
 
+  /**
+   * \returns \p true if the bounding box contains the given point,
+   * to within at least one of the given tolerance(s).  At least one
+   * tolerance should be set to greater than zero; otherwise use the
+   * other \p contains_point overload for efficiency.
+   *
+   * Relative tolerances are computed relative to the maximum finite
+   * extent of the bounding box, \p max_size()
+   */
+  bool contains_point (const Point &, Real abs_tol, Real rel_tol) const;
```

Comment:
```suggestion
  bool contains_point (const Point & p, Real abs_tol, Real rel_tol) const;
```

Reply:
Without it, I assume you don't actually use that argument in the implementation (which you do)